.github/workflows/ci.yml

```
@@ -0,0 +1,63 @@
+name: Sphinx contracts
+on:
+  merge_group:
+    push:
+      branches:
+        - "main"
+    pull_request:
+      types: [ opened, synchronize, reopened, ready_for_review ]
```

```suggestion
on:
  merge_group:
  push:
    branches:
      - "main"
  pull_request:
    types: [ opened, synchronize, reopened, ready_for_review ]
```
Fixing this should allow CI to trigger on this PR.